schemas/verifiablePresentation.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import verifiableCredential from './verifiableCredential.js';
+import {extend as _extend} from '../lib/helpers.js';
+import proof from './proof.js';
+import identifier from './identifier.js';
+import {klona} from 'klona';
+
+const schema = {
+  title: 'Verifiable Presentation',
+  type: 'object',
+  additionalProperties: true,
+  properties: {
+    '@context': {
+      type: 'array',
+      minItems: 1,
+      items: [
+        {type: 'string', const: 'https://www.w3.org/2018/credentials/v1'},
+        {anyOf: [{type: 'string'}, {type: 'object'}]}]
+    },
+    id: identifier(),
+    type: {
+      type: 'array',
+      minItems: 1,
+      items: {
+        type: 'string'
+      }
+    },
+    verifiableCredential: {
+      anyOf: [
+        verifiableCredential(),
+        {type: 'array', minItems: 1, items: verifiableCredential()}
+      ]
+    },
+    holder: identifier(),
```

Comment:
This can also be an object with an `id` field that is an identifier.

Reply:
Do we want to update the identifier validator to expect either a string or an object with an id or just create a new validator the builds on `identifier` and then re-use the new validator in `verifiableCredential` & `verifiablePresentation`? Also thanks for the review.

Reply:
I went with `jsonldIdentifier.js` here: https://github.com/digitalbazaar/bedrock-validation/pull/44/commits/f40ec654535b44934fcefc42e7580e5acbbdb181